tkgi/manifest.json

```diff
@@ -1,17 +1,17 @@
 {
-  "display_name": "Pivotal PKS",
+  "display_name": "Tanzu Kubernetes Grid Integrated",
```

Comment:
This can't change, but it shouldn't have any UI impact, just the public_title

Reply:
```suggestion
  "display_name": "Pivotal PKS",
```